src/api/user/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "user-microservice",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node server.js",
+    "dev": "nodemon server.js",
+    "emu": "firebase emulators:start --project telescope --only firestore",
+    "jest": "jest",
+    "test": "run-p emu jest"
+  },
+  "jest": {
+    "testEnvironment": "node",
+    "coveragePathIgnorePatterns": [
+      "/node_modules/"
+    ]
+  },
+  "dependencies": {
+    "@senecacdot/satellite": "^1.7.0",
+    "celebrate": "^13.0.4",
+    "express": "~4.16.1",
+    "firebase-admin": "^9.5.0"
```

Comment:
I don't think this is needed.

Reply:
`firebase-admin` is what persists data to the db